pkgs/top-level/php-packages.nix

```diff
@@ -518,10 +541,23 @@ lib.makeScope pkgs.newScope (self: with self; {
         { name = "pspell"; configureFlags = [ "--with-pspell=${aspell}" ]; }
         {
           name = "readline";
-          buildInputs = [ libedit readline ];
-          configureFlags = [ "--with-readline=${readline.dev}" ];
-          postPhpize = lib.optionalString (lib.versionOlder php.version "7.4") ''
```

Comment:
I have a patch that removes all these < 7.4 detrita, will submit it later so that this is not blocked any more.

Reply:
I guess we can merge this PR with your upcoming patch? Better wait for it I guess?

Reply:
I still need to do `generic.nix` and test it so I would not want to block this (and also https://github.com/fossar/nix-phps/pull/101) on it.